Specs/Number/Swedish/NumberModel.json

```diff
@@ -768,22 +768,6 @@
       }
     ]
   },
-  {
-    "Input": "femtio - två",
```

Comment:
I'd prefer if we had some normalization for this case into "femtio två" for the lookup, so we could still support it.